models.go

```diff
@@ -365,51 +371,119 @@ const (
 	DISABLED
 )
 
+// Logic is an enum type for policy logic
+type Logic string
+
+// Logic values
+const (
+	POSITIVE Logic = "POSITIVE"
+	NEGATIVE Logic = "NEGATIVE"
+)
+
 // PolicyRepresentation is a representation of a Policy
 type PolicyRepresentation struct {
 	Config           map[string]string `json:"config,omitempty"`
-	DecisionStrategy *DecisionStrategy `json:"decisionStrategy,omitempty"`
+	DecisionStrategy DecisionStrategy  `json:"decisionStrategy,omitempty"`
```

Comment:
Because then I can use it to set `DecisionStrategy` when creating a `PolicyRepresentation` or `PermissionRepresentation`.

Alternatively I could just make it a *string and not have enums.

Reply:
Ok, I see. Thank you for the explanation.